spec/models/candidate_interface/personal_details_form_spec.rb

```diff
@@ -96,16 +96,16 @@
   end
 
   describe '#date_of_birth' do
-    it 'returns nil for nil day/month/year' do
+    it 'returns an OpenStruct with nil values for nil day/month/year' do
       personal_details = CandidateInterface::PersonalDetailsForm.new(day: nil, month: nil, year: nil)
 
-      expect(personal_details.date_of_birth).to eq(nil)
+      expect(personal_details.date_of_birth).to eq(OpenStruct.new(day: nil, month: nil, year: nil))
```

Comment:
These specs are coupled to OpenStruct too. Could they use its public interface (`#month`, `#day` etc) instead?

Reply:
Resolved in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1003/commits/54c327762f421c982e7db498761bf5a4faae9107